dist/public.css

```diff
@@ -10,4 +10,25 @@ body {
   border-bottom: 2px solid;
 }
 
+.wp-site-blocks .site-banner {
```

Comment:
I missed this earlier but we ship all assets minified. You should be able to do this with `npm run build:production`

Reply:
Done.